spec/operators/take-spec.ts

```diff
@@ -14,7 +12,28 @@ describe('take operator', () => {
     testScheduler = new TestScheduler(observableMatcher);
   });
 
-  asDiagram('take(2)')('should take two values of an observable with many values', () => {
```

Comment:
IDK what the implications of removing this are.

Reply:
Nothing, really. I can be put back, but as of right now, we don't run the diagram generation anymore.

Reply:
We'll tackle these in another PR, cc @niklas-wortmann